openchs-server-api/src/main/java/org/openchs/web/RuleController.java

```diff
@@ -87,4 +87,29 @@ public RuleController(RuleService ruleService) {
             return ResponseEntity.badRequest().body(ruleResponseEntity);
         }
     }
+
+    @RequestMapping(value = "/web/visitrule", method = RequestMethod.POST)
+    ResponseEntity<?> visitScheduleRules(@RequestBody RequestEntityWrapper requestEntityWrapper) throws IOException, JSONException {
+        RuleResponseEntity ruleResponseEntity = null;
+        if(requestEntityWrapper.getRule().getWorkFlowType() != null) {
+            switch (requestEntityWrapper.getRule().getWorkFlowType().toLowerCase()) {
+                case "programenrolment":
```

Comment:
We should create an enum for WorkFlowType also so it's always from a fixed set of values.